contracts/src/v0.8/automation/dev/v2_2/AutomationRegistry2_2.sol

```diff
@@ -86,6 +85,7 @@ contract AutomationRegistry2_2 is AutomationRegistryBase2_2, OCR2Abstract, Chain
     UpkeepTransmitInfo[] memory upkeepTransmitInfo = new UpkeepTransmitInfo[](report.upkeepIds.length);
     uint16 numUpkeepsPassedChecks;
 
+    uint256 blocknumber = hotVars.chainModule.blockNumber();
```

Comment:
Is this expected to be present on all chains?

Reply:
chain module and block number are expected to be on all chains.